### PR TITLE
Add thank you and donation pages

### PR DESCRIPTION
**Pull request by bitigchi**
Fixes a couple of typos and joins lines for proper wrapping.

### CODE REVIEW

File: content/blog/pulkomandy/2021-06-03-haiku_activity_report_may_2021.md [hunk 1]
```diff
@@ -8,157 +8,93 @@ tags = []
...
-time. This will make the first run of Haiku boot a little bit faster.
+AGMS and PulkoMandy moved some work done on first boot of Haiku to compile time. This will make the first run of Haiku boot a little bit faster.
 
 <h3>Other platforms ports</h3>
 
-X512 has made major support on RISC-V support, with Haiku fully running in
-TinyEMU (no actual RISC-V hardware is supported yet). This work is currently
-being submitted and reviezed on Gerrit, some of the first patches have already
-been merged.
+X512 has made major support on RISC-V support, with Haiku fully running in TinyEMU (no actual RISC-V hardware is supported yet). This work is currently being submitted and revised on Gerrit, some of the first patches have already been merged.
```
Comment (pulkomandy) [inline content/blog/pulkomandy/2021-06-03-haiku_activity_report_may_2021.md:76]:
```suggestion
X512 has made major support on RISC-V support, with Haiku fully running in TinyEMU (no actual RISC-V hardware is supported yet). This work is currently being submitted and reviewed on Gerrit, some of the first patches have already been merged.
```

File: content/blog/pulkomandy/2021-06-03-haiku_activity_report_may_2021.md [hunk 2]
```diff
@@ -8,157 +8,93 @@ tags = []
 
 Hello, it's time for the May activity report!
 
+Before starting the report, I would like to thank our donors for their donations. Your donations help us cover our expenses and help us reach our goal to hire people to work on Haiku full-time. I would also like to thank all the community for their countless hours of effort of implementing new features, triaging bugs, translating, supporting other users, and spreading the Haiku word all around.
```
Comment (pulkomandy) [inline content/blog/pulkomandy/2021-06-03-haiku_activity_report_may_2021.md:11]: I am uncomfortable with the use of first person here, especially as this ends up on my blog and I didn't write this.

**Comment by leavengood** [comment]
For what it is worth I was planning to add a blog post or news item thanking donors with various information and updates. I have had delays in finishing the 2020 financial report and I wanted to try to wrap that up before I made the post. I should also finally get some donation tracking on the haiku-inc.org site. Wrapping all that up has delayed my post but I should have time this week.

File: content/blog/pulkomandy/2021-06-03-haiku_activity_report_may_2021.md [hunk 3]
```diff
@@ -8,157 +8,95 @@ tags = []
...
-time. This will make the first run of Haiku boot a little bit faster.
+AGMS and PulkoMandy moved some work done on first boot of Haiku to compile time. This will make the first run of Haiku boot a little bit faster.
 
 <h3>Other platforms ports</h3>
 
-X512 has made major support on RISC-V support, with Haiku fully running in
-TinyEMU (no actual RISC-V hardware is supported yet). This work is currently
-being submitted and reviezed on Gerrit, some of the first patches have already
-been merged.
+X512 has made major support on RISC-V support, with Haiku fully running in TinyEMU (no actual RISC-V hardware is supported yet). This work is currently being submitted and reviewed on Gerrit, some of the first patches have already been merged.
```
Comment (leavengood) [inline content/blog/pulkomandy/2021-06-03-haiku_activity_report_may_2021.md:76]:
```suggestion
X512 has made major progress on RISC-V support, with Haiku fully running in TinyEMU (no actual RISC-V hardware is supported yet). This work is currently being submitted and reviewed on Gerrit, some of the first patches have already been merged.
```

**Review by leavengood — COMMENTED** [review]
I think this makes this sentence sound a bit better.

**Pull request merged**